ush/launch_FV3LAM_wflow.sh

```diff
@@ -151,6 +151,19 @@ module load "${WFLOW_MOD_FN}" > /dev/null 2>&1 || print_err_msg_exit "\
 Loading of platform-specific module file (WFLOW_MOD_FN) for the workflow 
 task failed:
   WFLOW_MOD_FN = \"${WFLOW_MOD_FN}\""
+
+#
+#-----------------------------------------------------------------------
+#
+# Hack for Gaea since system python version is 3.4.10
```

Comment:
@MichaelLueken You are correct, it should read for Cheyenne!
I think best would be to update the system python version on Cheyenne to atleast v3.6. Will open an issue for that later.
The SRW app requirement is not clear because `environment.yaml` specifies python version of atleast 3.7 and in the code, there is a requirement of atleast 3.6 in `generate_workflow.py`. These seem to be requirements for the python version in coda specifically, not for the system python. The transition from shell to python workflow generation has imposed system python requirement of atleast version 3.6, so this need to be spelled out in the document. All platforms had system python version of >3.6 except Cheyenne that had `3.4.10`. This was the cause for the crontab deletion issue.

Reply:
I've opened up the issue here #402

Reply:
@danielabdi-noaa Thanks!